packages/mesos/README.md

```diff
@@ -1,6 +1,7 @@
 <H2>Patches to cherry-pick on top of open source Apache Mesos before building in DC/OS</h2>
 These commits can be found in the repository at <a href="https://github.com/mesosphere/mesos/">https://github.com/mesosphere/mesos/</a>:
-<li>[c81bea7092bea8a2912237abdcab027e67c1dd91] Set LIBPROCESS_IP into docker container.
-<li>[5720037ac5921b675d59b670ecf6ea24cfbdf317] Changed agent_host to expect a relative path.
-<li>[38d4c3022e22eb98006ad50c3e9c5e115d5f14e3] Mesos UI: Change paths, pailer, and logs to work with a reverse proxy.
-<li>[ffea5a593c030dbdfced57942a2ca5c6e447fcf3] Revert "Fixed the broken metrics information of master in WebUI."
+<li>[e6b2a0a8203f2507ef5ae4e3198ef649384a356d] Set LIBPROCESS_IP into docker container.
```

Comment:
These hashes don't seem to match the related commits, eg.
e6b2a0a8203f2507ef5ae4e3198ef649384a356d
vs.
https://github.com/mesosphere/mesos/commit/42e4e19d0d7e54d5e361e1dd2d6afbd24d1928d1